scripts/tokenomics-emulation/emulations/validators.emulation.ts

```diff
@@ -0,0 +1,44 @@
+import { IEmulation } from "./emulation.interface";
+import Network from "../network";
+import Accounts from "../accounts";
+import Validator from "../validator";
+import fs from "fs";
+
+class ValidatorsEmulation implements IEmulation {
```

Comment:
Let's rename to `AddValidatorsEmulation`

Reply:
Renamed